packages/dds/ordered-collection/src/consensusOrderedCollection.ts

```diff
@@ -278,16 +279,18 @@ export class ConsensusOrderedCollection<T = any>
      */
     protected async loadCore(storage: IChannelStorageService): Promise<void> {
         assert(this.jobTracking.size === 0);
-        const rawContentTracking = await storage.read(snapshotFileNameTracking);
+        const blob = await storage.readBlob(snapshotFileNameTracking);
+        const rawContentTracking = blobToString(blob);
         if (rawContentTracking !== undefined) {
-            const content = this.deserializeValue(fromBase64ToUtf8(rawContentTracking), this.serializer);
```

Comment:
Can we remove deserializeValue() in favor of deserializeBlob() workflow?
Is rawContentTracking !== undefined check doing anything useful? I think blobToString should never return undefined, no?